package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "UMA Protocol Smart Contracts",
   "scripts": {
-    "prettier": "npm run prettier_vanilla -- --write",
+    "prettier": "$(npm bin)/eslint --fix ./core/**/*.js --quiet && npm run prettier_vanilla -- --write",
```

Comment:
Doesn't this mean that this doesn't get checked in ci? If we want it to be checked, we probably need to make it part of prettier_vanilla?

Also, thoughts on somehow putting the prettier commands in a bash script since they're getting pretty long? Not sure if npm will play nice with a bash call.

Reply:
Yes, this is correct. it does not run in CI. it is currently only used to add the correct commenting style when prettier is run. Adding it CI can be done as well.